aspnetcore/signalr/streaming/samples/2.2/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We've been using two-space indents, but I get it if this was generated this way automagically by something.